frontend/src/container/PlannedDowntime/DropdownWithSubMenu/DropdownWithSubMenu.styles.scss

```diff
@@ -0,0 +1,119 @@
+.options {
+	width: 100%;
+}
+
+.option {
+	padding: 8px 10px;
+	cursor: pointer;
+	overflow: auto;
+}
+
+.option:hover {
+	background-color: var(--bg-slate-200);
+}
+
+.submenu-container {
+	position: absolute;
+	top: 0;
+	right: 50%;
+	z-index: 1;
+	background-color: var(--bg-ink-400);
+	border: 1px solid var(--bg-slate-500);
+	max-height: 300px;
+	overflow-y: auto;
+	width: 160px;
+	display: flex;
+	flex-direction: column;
+	gap: 6px;
+	padding: 12px;
+	border-radius: 4px;
+}
+
+.dropdown-submenu {
+	input[type='text'] {
```

Comment:
can we add classname here ? any specific reason of using this selector ?

Reply:
done used classname